src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <div
+      className={classNames('todo', {
+        completed,
+      })}
+      key={id}
```

Comment:
The 'key' attribute should be placed on the component that is being mapped, not on the individual elements inside the component. Remove the 'key' attribute from this div.